client/state/themes/selectors.js

```diff
@@ -325,12 +325,18 @@ export function isWporgTheme( state, themeId ) {
 /**
  * Whether a theme is present on WordPress.com.
  *
+ * Returns false if themeId is installed on (jetpack) siteId, since
+ * installed themes take priority. For example, this can be the case
+ * with themes available on both wpcom and .org such as 'twentysixteen'.
+ *
  * @param  {Object}  state   Global state tree
  * @param  {Number}  themeId Theme ID
+ * @param  {Number}  siteId  Site ID
  * @return {Boolean}         Whether theme available on WordPress.com
  */
-export function isWpcomTheme( state, themeId ) {
-	return !! getTheme( state, 'wpcom', themeId );
+export function isWpcomTheme( state, themeId, siteId ) {
```

Comment:
This feels like it stopped to have the original meaning. 
> Whether a theme is present on WordPress.com.
So this function can return False even if the theme is present on wpcom. 
Maybe lets have a second function or rename this one.